web/static/css/styles.css

```diff
@@ -164,6 +164,10 @@ header {
   margin-bottom: 0;
 }
 
+.ca-gov-icon-check-fill {
+  color: #336C39;
```

Comment:
This is the value from Figma. I checked over the variables coming from the CA State Web Template CSS and didn't see one with this hex value, so just defined it here.

<img width="1920" height="1152" alt="Screenshot from 2025-08-21 13-54-32" src="https://github.com/user-attachments/assets/6c171277-6ca2-4c39-8c55-7495de12e1a3" />